src/data/markdown/docs/01 guides/02 Using k6/01 HTTP requests.md

```diff
@@ -84,9 +87,11 @@ Below you can see how a test result data point (the duration of an HTTP request)
     "time": "2017-06-02T23:10:29.52444541+02:00",
     "value": 586.831127,
     "tags": {
+      "expected_response": "true",
       "group": "",
       "method": "GET",
       "name": "http://test.k6.io",
+      "scenario": "",
```

Comment:
As I said above, this is wrong. Run `k6 run --out json=results.json 'https://raw.githubusercontent.com/loadimpact/k6/master/samples/http_get.js' && jq '. | select(.type=="Point" and .metric == "http_req_duration")' results.json`, you'd get:
```js
{
  "type": "Point",
  "data": {
    "time": "2021-03-16T11:28:25.408816519+02:00",
    "value": 409.892874,
    "tags": {
      "expected_response": "true",
      "group": "",
      "method": "GET",
      "name": "https://test-api.k6.io/",
      "proto": "HTTP/2.0",
      "scenario": "default",
      "status": "200",
      "tls_version": "tls1.2",
      "url": "https://test-api.k6.io/"
    }
  },
  "metric": "http_req_duration"
}
```